README.md

```diff
@@ -1,30 +1,34 @@
 linter-flake8
 =============
 
-linter-flake8 is a [flake8](https://pypi.python.org/pypi/flake8 provider for [Linter](https://github.com/AtomLinter/Linter).
+linter-flake8 is a [flake8](https://pypi.python.org/pypi/flake8) provider for [Atom-Linter](https://github.com/AtomLinter/Linter).
 
 ![img](https://cloud.githubusercontent.com/assets/4278113/8768482/52f975c6-2e3f-11e5-87e4-27c8359fd36c.gif)
 
 ### Requirements
-Linter package must be installed in order to use this plugin. If it's not
-installed, please follow the instructions [here](https://github.com/AtomLinter/Linter).
+The `Linter` package must be installed in order to use this package. If it's not installed, please follow the [Linter instructions](https://github.com/AtomLinter/Linter).
 
 ### Installation
-To use a plugin, you must ensure that `flake8` is installed on your
-system.
-
-Install [flake8](https://pypi.python.org/pypi/flake8) by typing the following
-in a terminal:
+First, ensure that [flake8](https://pypi.python.org/pypi/flake8) is installed on your system. If not, install it by typing the following in a terminal:
    ```
    pip install flake8
    ```
 
-Install plugin by typing:
+Install package by typing:
    ```
```

Comment:
I usually put `ShellSession` here ([source](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L3172)), but it isn't important.